src/components/Pressable/GenericPressable/index.native.tsx

```diff
@@ -0,0 +1,20 @@
+import React, {ForwardedRef, forwardRef} from 'react';
+import {View} from 'react-native';
+import GenericPressable from './BaseGenericPressable';
+import PressableProps from './types';
+
+function NativeGenericPressable(props: PressableProps, ref: ForwardedRef<View>) {
+    return (
+        <GenericPressable
+            focusable
+            accessible
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...props}
+            ref={ref}
+        />
```

Comment:
I think `accessibilityHint={props.accessibilityHint || props.accessibilityLabel}` is missing there

Reply:
@robertKozik it was added previously by you, do we need this?

Reply:
It was a precautionary measure on my part. If no hint was provided, we would default it to the required label. So it's not strictly necessary, but I'll re-add it to ensure it works the same as the current version in the main branch.

Reply:
Yep, it is needed. Good catch!